### PR TITLE
1243 comboinput placeholder translated

### PR DESCRIPTION
**Pull request by mvniekerk**

### CODE REVIEW

File: lib/util/translation.ts [hunk 1]
```diff
@@ -20,6 +20,9 @@ export function translatedValue<T>(i18nKey: TranslatedValueOrKey<T>): ITranslate
 }
 
 export function translateItem<T>(i: TranslatedValueOrKey<T>): string {
+  if (!i) {
```
Comment (melissapalmer) [inline lib/util/translation.ts]: What is i ?   
Can this variable name not be more descriptive

File: package.json [hunk 2]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "@grindrodbank/spoon-ui",
-  "version": "0.6.9",
+  "version": "0.6.11",
```
Comment (melissapalmer) [inline package.json]: Surely next version after 9 is 10

File: lib/util/translation.ts (hunk 1 above)
Comment (mvniekerk) [reply lib/util/translation.ts]: The i is the only input, and is immediately transformed to model variables that have more descriptive names.

File: package.json (hunk 2 above)
Comment (mvniekerk) [reply package.json]: Made an error after bumping the version

File: lib/util/translation.ts (hunk 1 above)
Comment (melissapalmer) [reply lib/util/translation.ts]: mm

**Comment by philipstaffordwood** [comment]
Looks good.

**Pull request merged**